### ISSUE TITLE
 training parameters for libri1k

### ISSUE THREAD

**Issue by slbinilkumar** [SeanNaren/deepspeech.pytorch#132]
For libri1k test-clean whats your validation set ? whats your training parameters? for getting this result libri1k | test-clean | 5x800gru | 11.546 | 3.538 .How much time taken for it to converge? How many epochs you trained for achieving this?
Corpus | Test Set | Network | WER | CER
-- | -- | -- | -- | --
an4 | an4-test | 5x800gru | 10.521 | 4.772
libri1k | libri-val | 5x800gru | 20.758 | 7.787
libri1k | libri-test | 5x800gru | 22.088 | 8.194
libri1k | test-clean | 5x800gru | 11.546 | 3.538
libri1k | test-other | 5x800gru | 31.813 | 12.483

**Comment by ryanleary** [comment]
Closing. Answered your question in #85.
